src/mocked-env-spec.js

```diff
@@ -189,4 +203,68 @@ describe('mocked-env', () => {
       restore()
     })
   })
+
+  describe('restoring process.env', () => {
+    let restore
+    let defaultEnv
+
+    beforeEach(() => {
+      defaultEnv = R.clone(process.env)
+      process.env.BANG = 'bash'
+      restore = mockedEnv(
+        {
+          FOO: 'foo',
+          BAR: 'bar'
+        },
+        { restore: true }
+      )
+    })
+
+    it('has default env, BANG, FOO and BAR', () => {
```

Comment:
hmm, I am not sure this is the best test for this. We want to check "restore" right? So we probably better do
```
const restore = mockedEnv({...}, {restore: true})
process.env.BANG = 'bash'
restore()
la(process.env.BANG === undefined, 'did not restore process.env')
```

what do you think?

Reply:
That test only verifies that the environment variable that was present before mocking the environment was available during the mock when "restore" is given.

The later test "pass 'restore' in options" actually tests the case you're referring to where it changes the variable via process.env after mocking then verifies it restores it after.

Reply:
I see what you mean, merging your PR now